src/sentry/monitors/models.py

```diff
@@ -387,20 +387,54 @@ class MonitorCheckIn(Model):
     monitor = FlexibleForeignKey("sentry.Monitor")
     monitor_environment = FlexibleForeignKey("sentry.MonitorEnvironment", null=True)
     location = FlexibleForeignKey("sentry.MonitorLocation", null=True)
+    """
+    XXX(epurkhiser): Currently unused
+    """
     status = BoundedPositiveIntegerField(
-        default=CheckInStatus.UNKNOWN, choices=CheckInStatus.as_choices(), db_index=True
+        default=CheckInStatus.UNKNOWN,
+        choices=CheckInStatus.as_choices(),
+        db_index=True,
     )
-    config = JSONField(default=dict)
+    """
+    The status of the check-in
+    """
     duration = BoundedPositiveIntegerField(null=True)
+    """
+    The total number in milliseconds that the check-in took to execute. This is
+    generally computed from the difference between the open and close check-in.
+    """
     date_added = models.DateTimeField(default=timezone.now, db_index=True)
+    """
+    Represents the time the checkin was made. This CAN BE back-dated in some
+    cases, and does not necissarily represent the insertion time of the row in
+    the database.
+    """
     date_updated = models.DateTimeField(default=timezone.now)
-    attachment_id = BoundedBigIntegerField(null=True)
-    # Holds the time we expected to receive this check-in without factoring in margin
+    """
+    Last update time for the check-in. Can be used to extend the lifetime of a
```

Comment:
this isn't true. this is the last time we received an `IN_PROGRESS` status for the check-in. the second sentence is true

Reply:
oh do we not update this when we close a check-in either?